README.md

```diff
@@ -0,0 +1,9 @@
+# Cockroach Protocol Buffers
+
+This repo contains protobuf definitions extracted from the [main cockroach repo](https://github.com/cockroachdb/cockroach). They have been cleaned by removing all [`gogoproto`](https://github.com/gogo/protobuf) extensions to facilitate compilation in other languages.
+
+The directory `cockroach/sql/driver` contains all the `.proto` files expected to be used by clients and has no dependencies. For client development, it is generally easier to compile this directory alone instead of building all protos in the package. The file `cockroach/multiraft/rpc.proto` is particularly tricky to compile due to its dependency on protos defined by `etcd` (and thereby indirectly retaining a dependency on `gogoproto`).
```

Comment:
It's mostly speculative; the only concrete use case I'm anticipating is a wireshark plugin that can understand our RPCs. I think that with go 1.5's shared library options it's now possible to do that in go, but in case it's not we may need other protos to be accessible from other languages.